src/lib/stitching/gravity/schema.ts

```diff
@@ -0,0 +1,41 @@
+import { createGravityLink } from "./link"
+import {
+  makeRemoteExecutableSchema,
+  transformSchema,
+  RenameTypes,
+  FilterRootFields,
+} from "graphql-tools"
+import { readFileSync } from "fs"
+
+export const executableGravitySchema = async () => {
+  const gravityTypeDefs = readFileSync("src/data/gravity.graphql", "utf8")
+
+  const gravityLink = createGravityLink()
+  const schema = await makeRemoteExecutableSchema({
+    schema: gravityTypeDefs,
+    link: gravityLink,
+  })
+
+  // Remap the names of certain types from Gravity to fit in the larger
+  // metaphysics ecosystem.
+  const remap = {
+    Artist: "GravityArtist",
+    Artwork: "GravityArtwork",
+    Partner: "GravityPartner",
+  }
```

Comment:
IMO we shouldn’t have implementation details such as in which service data lives in the schema. Why are we pulling these in atm? Shouldn’t we be pulling these in when we make these the canonical `Artist`, `Artwork`, and `Partner`?